src/assignment/components/AssignmentActions.tsx

```diff
@@ -44,28 +50,13 @@ const AssignmentActions: FunctionComponent<AssignmentActionsProps> = ({
 	overflow,
 	duplicate,
 	remove,
-	shareWithPupilsProps,
+	share,
```

Comment:
there are 3 types of sharing in the application
i would prefer to make a clear distiction
maybe we can rename it to this?
```suggestion
	shareWithColleguesOrPupilProps,
```